### PR TITLE
netaddr: return parseIPError for all ParseIP errors

### PR DESCRIPTION
**Pull request by nwt**
This ensures ParseIP errors always begin with `ParseIP(<input>): ` when
converted to strings.

### CODE REVIEW

**Review by mdlayher — APPROVED** [review]
LGTM thanks!

**Comment by nwt** [comment]
Can I nudge this toward main?

**Comment by mdlayher** [comment]
I was waiting for a second review as I am not the primary user of this repo, but this seems trivially correct to me so I'll merge. Thanks.

**Comment by nwt** [comment]
Thanks, @mdlayher!

**Pull request merged**
